> go list -mod=readonly -m all

Without fix

```
[🎩︎]mrajanna@fedora ceph-csi $]make mod-check 
cephcsi image settings: quay.io/cephcsi/cephcsi version canary
running: go mod verify
all modules verified
running: go list -mod=readonly -m all
go: k8s.io/dynamic-resource-allocation@v0.0.0: invalid version: unknown revision v0.0.0
make: *** [Makefile:110: mod-check] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -106,6 +106,8 @@ go-test-api: check-env
 mod-check: check-env
 	@echo 'running: go mod verify'
 	@go mod verify && [ "$(shell sha512sum go.mod)" = "`sha512sum go.mod`" ] || ( echo "ERROR: go.mod was modified by 'go mod verify'" && false )
+	@echo 'running: go list -mod=readonly -m all'
+	@go list -mod=readonly -m all 1> /dev/null
 
 scripts/golangci.yml: scripts/golangci.yml.in
 	rm -f scripts/golangci.yml.buildtags.in

--- a/go.mod
+++ b/go.mod
@@ -186,6 +186,7 @@ replace (
 	k8s.io/controller-manager => k8s.io/controller-manager v0.26.1
 	k8s.io/cri-api => k8s.io/cri-api v0.26.1
 	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.26.1
+	k8s.io/dynamic-resource-allocation => k8s.io/dynamic-resource-allocation v0.26.1
 	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.26.1
 	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.26.1
 	k8s.io/kube-proxy => k8s.io/kube-proxy v0.26.1

